conference_101719.tex

```diff
@@ -0,0 +1,151 @@
+
+%Since this scheme mainly utilizes the training detection technique and uses a relatively static threshold to differentiate legitimate and malicious routes, it needs to precollect the information of the target Wi-Fi network.
+%Thus this scheme is not useful in the networks whose situsation may have significantly change or which some inexperienced users connect to.
+
+
+\section{ATTACK MODEL AND PREVIOUS SCHEME}
+\subsection{Attack Model}
+In an evil twin attack, the adversary sets up his AP using the SSID of the targeted Wi-Fi network and also MAC address cloned from one of the APs in the network.
+As a result, a user's device receives SSID broadcast from both the lAP and the rAP, but it cannot differentiate between these APs.
+The user's device simply assumes that both the APs are legitimate and associates with the one that has a higher RSSI value.
+We assume the model that a rAP relays WLAN traffic between a legitimate AP providing Internet connectivity and a user's device, and may act as a man-in-the-middle-attack.
+By avoiding to use mobile Internet access, e.g., 3G/4G, the adversary can evade detection with ISP names or Global IP addresses\cite{rtt}.
+Also, we assume the adversary launched hardware-based APs which cannot be detected accurately by existing schemes.
+Thus, it cannot be detected because hardware-based APs do not cause a computational delay by the software bridging.
+
+\subsection{Previous Scheme}
+\subsubsection{Overview of the Previous Scheme}
+The main idea of the previous scheme \cite{previous} is that the adversary needs to use two distinct communication channels on the path from user's device to the DNS server to avoid channel interference each other: one for the path between lAP and rAP and the other for that betweeen rAP and user's device.
+Thus, from the perspective of the user's device, there exists another channel on the route different from a channel the device uses with the connected AP. 
+Since the throughput value is dependent on the traffic on the path, the channel which is used between a lAP and rAP and not able to be seen directly from the user's device can be detected as a decline in the throughput by saturating that path.
+For example, when a user's device is using channel 1 with the targeted AP which cannot be judged to be legitimacy, the user-side device introduced for intentional channel interference transmit several packets with all the channel except channel 1 to saturate the path.
+If there exists the other channel on the route, the decline in the throughput can be observed by the user's device.
+
+\subsubsection{Shortcoming of the Previous Scheme}
+Although the previous scheme is successful in the detection for hardware-based rAP in the experimental environment, it cannot detect accurately in the real world because of various reasons such as collisions, network topology changes, unintentional interference and so on.
```

Comment:
```suggestion
Although the previous scheme is successful in the detection for hardware-based rAP in the experimental environment whose traffic is stable, it cannot detect accurately in the real world because of various reasons such as collisions, network topology changes, unintentional interference and so on.
```